scripts/openshiftci-presubmit-e2e.sh

```diff
@@ -9,7 +9,9 @@ export CI="openshift"
 export TIMEOUT="30m"
 make configure-installer-tests-cluster
 make bin
-export PATH="$PATH:$(pwd)"
+mkdir -p $GOPATH/bin
+go get -u github.com/onsi/ginkgo/ginkgo
```

Comment:
This adds it to just OpenShift-CI. It won't be needed on travis?
We have `make goget-tools` that should be used for this.

Reply:
Good point. 
Done